src/index.html

```diff
@@ -16,17 +19,33 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-bar__body">
```

Comment:
The `class` attribute value `search-bar__body` suggests that BEM methodology is being used, which is good for CSS maintainability. However, ensure that `search-bar__body` represents the meaning of the content and is not merely styling or tag names.